examples/graphics-2d/README.md

```diff
@@ -16,6 +16,10 @@ Choose a run configuration for an appropriate target in Android Studio and run i
 
 `./gradlew desktopApp:run`
 
-## Run native on MacOS
-Choose **shared[macosX64]** or **shared[macosArm64]** configuration in IDE and run it.
+## Run experimental JS on Browser
```

Comment:
Let's remove this section. We don't write public docs for the Web JS target. The official web target uses wasm, and the JS target is an experimental implementation and a road to the wasm web implementation

Reply:
We may replace JS to WASM, when we merge WASM to master. I would leave JS target

Reply:
I also think it's better to leave this section in the README. This is an experimental target, but it works. 
But, I suggest to chage title. Let's call it simple: **Run experimental on Browser** (without JS mentions)

@igordmn What do you think?

Reply:
@igordmn Done, removed